scripts/target/mame/mess.lua

```diff
@@ -2647,6 +2648,11 @@ files {
 	MAME_DIR .. "src/mame/machine/psion_pack.h",
 }
 
+createMESSProjects(_target, _subtarget, "controlid")
+files {
+	MAME_DIR .. "src/mame/drivers/controlid.cpp",
```

Comment:
Could you please add this to the skeleton target or something else where it kind of fits rather than adding another single-driver project?  We have too many of these as it is.